cloudprem/compute/scaling.tf

```diff
@@ -0,0 +1,85 @@
+resource "aws_iam_policy" "cluster_autoscaler" {
```

Comment:
There's at least 3 different resources of different kinds with the same name `cluster_autoscaler` here. That can be a little confusing in state and plan output. Maybe we can name these something like `cluster_as_policy`?